vars/perfRegressionParallelPipelinebyRegion.groovy

```diff
@@ -0,0 +1,104 @@
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_arm64', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_arm64-write', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_x86_64', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-simple-query-weekly-microbenchmark_x86_64-write', region: 'us-east-1', versions: ['2024.1', '2024.2', '2025.1', '2025.2', 'master'], sub_tests: ['microbenchmark'], scheduler: 'master-vnodes'],
+                            [job_name: 'scylla-master/perf-regression/perf-regression-predefined-throughput-steps-sanity-vnodes', region: '', versions: [], sub_tests: [], scheduler: ''],
+                            // Tablets
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-tablets', region: 'us-east-1', versions: ['2025.1', '2025.2', 'master'], sub_tests: ['test_read_gradual_increase_load', 'test_mixed_gradual_increase_load'], scheduler: 'master-tablets'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-predefined-throughput-steps-write-tablets', region: 'us-east-1', versions: ['2025.1', '2025.2', 'master'], sub_tests: ['test_write_gradual_increase_load'], scheduler: 'master-tablets'],
+                            // One in 3 weeks
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-during-rolling-upgrade-tablets', region: 'eu-west-2', versions: ['2025.1', '2025.2', 'master'], sub_tests: ['test_mixed_gradual_increase_load'], scheduler: 'master-tablets'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-with-nemesis-tablets', region: 'eu-west-3', versions: ['2025.1', '2025.2'], sub_tests: ['test_read_gradual_increase_load', 'test_mixed_gradual_increase_load'], scheduler: 'master-tablets'],
+                            [job_name: 'scylla-enterprise/perf-regression/scylla-enterprise-perf-regression-latency-650gb-with-nemesis-tablets', region: 'eu-west-3', versions: ['master'], sub_tests: ['test_mixed_gradual_increase_load'], scheduler: 'master-tablets'],
```

Comment:
shouldn't it would be clearer to put `'master-tablets` into versions ? and when `matrix_storage` it replaces the version for test selection ?

so we don't need to specify master in the version list ?

Reply:
@fruch  I did not want to mix up Scylla version with test type selector